packages/compass-sidebar/src/components/sidebar.tsx

```diff
@@ -219,6 +220,7 @@ const MappedSidebar = connect(mapStateToProps, {
   updateAndSaveConnectionInfo,
   toggleIsGenuineMongoDBVisible,
   setIsExpanded,
+  setConnectionIsCSFLEEnabled,
```

Comment:
Looks like we still have the `sidebar-toggle-csfle-enabled` event being emitted here, should we remove that and use this function?

Reply:
Yeah, thanks for catching this … :see_no_evil: